dist/monet.js

```diff
@@ -1047,9 +1009,9 @@
         };
     }
     function addFunctorOps(type) {
-        if (type.prototype.map == null) {
+        if (!type.prototype.map) {
```

Comment:
How about `if(!isFuncton(type.prototype.map)) {` ? It will follow conventions from methods above (I guess).